glustercli/cmd/root.go

```diff
@@ -96,6 +97,19 @@ var (
 	flagTimeout    uint
 )
 
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print glusterd and gluster-cli version",
```

Comment:
Do we need the argument validation here (number of argument)?

Reply:
@rafikc30 I will be updating this PR. Thinking to remove version as an argument.